internal/exec/util/passwd.go

```diff
@@ -171,13 +172,20 @@ func (u Util) AuthorizeSSHKeys(c types.PasswdUser) error {
 		if !strings.HasSuffix(ks, "\n") {
 			ks = ks + "\n"
 		}
-
+		var path string
 		if distro.WriteAuthorizedKeysFragment() {
-			err = writeAuthKeysFile(usr, filepath.Join(usr.HomeDir, ".ssh", "authorized_keys.d", "ignition"), []byte(ks))
+			path = filepath.Join(usr.HomeDir, ".ssh", "authorized_keys.d", "ignition")
+			err = writeAuthKeysFile(usr, path, []byte(ks))
 		} else {
-			err = writeAuthKeysFile(usr, filepath.Join(usr.HomeDir, ".ssh", "authorized_keys"), []byte(ks))
+			path = filepath.Join(usr.HomeDir, ".ssh", "authorized_keys")
+			err = writeAuthKeysFile(usr, path, []byte(ks))
+		}
+		if err == nil {
+			journal.Send(fmt.Sprintf("ssh authorized keys were provided via ignition for user:%s", c.Name), journal.PriInfo, map[string]string{
```

Comment:
I wonder if this should be a generic "user added" message, and we include a count of authorized keys?  That way we are guaranteed to get at least one message for FCOS systems because we have the default Ignition that adds the `core` user.

Reply:
IOW we always write "user foo added" in the journal and in the data we include something like `IGNITION_SSH_AUTH_KEYS_NUM: X`, where X is the number of authorized keys, in the structured data?. Typically this would be 1, sometimes 0, sometimes more.

The benefit would be that we always are able to find one instance of this message. Though I'm not sure we really need to always find an instance of this message do we?

Reply:
unrelated to the previous discussion here... one possible suggestion for better message: `"wrote ssh authorized keys file for user: %s"` - The "ignition" part will be obvious because jourrnal will prefix the log line with ignition anyway.